qa/suites/fs/multiclient/clusters/three_clients.yaml

```diff
@@ -1,5 +1,5 @@
 roles:
-- [mon.a, mon.b, mon.c, mgr.x, mds.a, osd.0, osd.1, osd.2]
+- [mon.a, mon.b, mon.c, mgr.x, mds.a, osd.0, osd.1, osd.2, osd.4]
```

Comment:
should be osd.3

http://pulpito.ceph.com/pdonnell-2017-10-23_19:01:45-fs-wip-pdonnell-testing-20171019-testing-basic-smithi/1765589